frontend/src/components/WofButton.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <button :class="`button button--${type}`" @click="onClick()">
+    <div v-if="this.$slots.icon" :class="this.$slots.default ? 'button--icon' : ''"><slot name="icon"/></div>
+    <slot/>
+  </button>
+</template>
+
+<script>
+export default {
+  name: "WofButton",
+  props: {
+    type: {
+      type: String,
+      required: true,
+      default: "accent",
+      validator(value) {
+        return ["accent", "positive", "warning", "error", "disabled"].includes(value)
+      }
+    }
+  },
+  methods: {
+    onClick: function(event) {
+      this.$emit("click", event)
+    }
+  }
+}
+</script>
+
+<style lang="less">
+@import './common.less';
+
+.button {
+  border: none;
+  height: min-content;
+  text-align: center;
+  align-items: center;
+  display: flex;
+  border-radius: 1.25rem;
+  padding-top: 0.563rem;
+  padding-bottom: 0.563rem;
+  padding-left: 1rem;
+  padding-right: 1rem;
```

Comment:
Instead of these all paddings, just use "padding: 0.5em 1em;"

Reply:
done